packages/client/components/InvitationLink.tsx

```diff
@@ -1,16 +1,28 @@
 import React from 'react'
-import {createFragmentContainer} from 'react-relay'
+import {PreloadedQuery, usePreloadedQuery} from 'react-relay'
 import graphql from 'babel-plugin-relay/macro'
-import {InvitationLink_massInvitation} from '../__generated__/InvitationLink_massInvitation.graphql'
+import {InvitationLinkRootQuery} from '../__generated__/InvitationLinkRootQuery.graphql'
 import TeamInvitationMeetingAbstract from './TeamInvitationMeetingAbstract'
 import InvitationLinkDialog from './InvitationLinkDialog'
 
 interface Props {
-  massInvitation: InvitationLink_massInvitation
+  queryRef: PreloadedQuery<InvitationLinkRootQuery>
 }
 
 function InvitationLink(props: Props) {
-  const {massInvitation} = props
+  const {queryRef} = props
+  const data = usePreloadedQuery<InvitationLinkRootQuery>(
+    graphql`
+      query InvitationLinkRootQuery($token: ID!) {
+        massInvitation(token: $token) {
+          ...InvitationLinkDialog_massInvitation
+        }
+      }
+    `,
+    queryRef,
+    {UNSTABLE_renderPolicy: 'full'}
```

Comment:
Fun fact, I googled what that means, this was my first result: https://www.parabol.co/blog/upgrade-react-to-graphql-relay-hooks/ Kudos @mattkrick 
TL;DR: we need this until we get rid of `createFragmentContainer`